packages/libs/error-stack/src/report.rs

```diff
@@ -617,6 +647,23 @@ impl<Context> fmt::Debug for Report<Context> {
     }
 }
 
+#[cfg(feature = "std")]
+impl<Context> std::process::Termination for Report<Context>
+where
+    Context: fmt::Debug,
+{
```

Comment:
The `Debug` bound is not required. Or do you have a specific reason why you added this? If so, maybe `Context` (the trait) is a better bound?

```suggestion
impl<Context> std::process::Termination for Report<Context> {
```

Reply:
I think I initially did this to ensure a debug output of context but realized it wasn't necessary since the wrapping `Report` should have all relevant information.